frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -2053,6 +2142,77 @@ mod tests {
 		});
 	}
 
+	#[test]
+	fn emergency_throttling_works() {
+		ExtBuilder::default()
+			.onchain_fallback(false)
+			.phases(10, 10)
+			.build_and_execute(|| {
+				MultiPhase::rotate_round();
```

Comment:
Why do we start with this?

Reply:
No special reason, removing it will work as expected.